source/MaterialXGraphEditor/Graph.cpp

```diff
@@ -1869,7 +1869,6 @@ void Graph::copyInputs()
                         }
 
                         copyNode->inputPins[count]->setConnected(true);
-                        copyNode->inputPins[count]->_input->removeAttribute(mx::ValueElement::VALUE_ATTRIBUTE);
```

Comment:
Just removing the value can actually leave a state of no value and no connection.